### PR TITLE
Add support for m5n/m5dn/r5n instances

### PR DESCRIPTION
**Pull request by Jeffwan**
*Issue #, if available:*
#653 

This is a follow up PR of #656. I just notice there're few other instance releases together with r5dn. We should add support for them together. 

*Description of changes:*
Add eni and maximum ips for m5n/m5dn/r5n instances

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mogren** [comment]
> I wish there was a way to automate/generate this information :)

Yeah, like an EC2 API that would give us this info....

File: pkg/awsutils/vpc_ip_resource_limit.go [hunk 1]
```diff
@@ -148,6 +148,22 @@ var InstanceENIsAvailable = map[string]int{
 	"m5d.16xlarge":  15,
 	"m5d.24xlarge":  15,
 	"m5d.metal":     15,
+	"m5dn.large":    3,
```
Comment (mogren) [inline pkg/awsutils/vpc_ip_resource_limit.go]: We need the number of IPs per ENI for all types as well in this map:
```
var InstanceIPsAvailable = map[string]int{
```

Comment (Jeffwan) [reply pkg/awsutils/vpc_ip_resource_limit.go]: Ah, I forget to add this part. Just add to the list

**Comment by Jeffwan** [comment]
Maybe we can run some utility to parse page and auto generate a go file?
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html

Instead of adding instance types, we should generate full snapshot every time.

**Comment by mogren** [comment]
@Jeffwan we do have a script like that already, but it needs some updates 🙂

File: pkg/awsutils/vpc_ip_resource_limit.go (hunk 1 above)
Comment (mogren) [reply pkg/awsutils/vpc_ip_resource_limit.go]: Thanks!

**Pull request merged**